radis/levels/partfunc.py

```diff
@@ -487,23 +493,39 @@ def at_noneq(
 
         # ... mode: Trot, Tvib, no overpopulation, no vib band details
         else:  # slightly faster, but doesnt return nvib nor Qvib
-
-            g = gvib * grot
-
-            if vib_distribution == "boltzmann" and rot_distribution == "boltzmann":
-                nQ = g * exp(-hc_k * (df.Evib / Tvib + df.Erot / Trot))
-            elif vib_distribution == "treanor" and rot_distribution == "boltzmann":
-                nQ = g * exp(-hc_k * (df.Evib_h / Tvib + (df.Evib_a + df.Erot) / Trot))
+            if fast_sum:
+                rtol, N = 1e-15, 10000  # number found after a few tests on CO2(iso=1)
```

Comment:
here you're overriding the default arguments of partial_partition_sum_nargs. Should add these as default values directly in the function.